Jenkinsfile.release

```diff
@@ -183,20 +189,24 @@ spec:
         environment {
           AZURE_STORAGE_ACCOUNT = 'overninjenkinsreleases'
           AZURE_STORAGE_KEY = credentials('release-storage-account-key')
-          JENKINS_WAR = "${ env.WORKSPACE + '/war/target/jenkins.war' }"
-          JENKINS_ASC = "${ env.WORKSPACE + '/war/target/jenkins.asc' }"
-          AZURE_SHARE_NAME = "war-rc"
+          JENKINS_WAR = "$WORKSPACE/war/target/jenkins.war"
+          JENKINS_ASC = "$WORKSPACE/war/target/jenkins.asc"
+          AZURE_SHARE_NAME = 'war-rc'
         }
         steps {
           container('azure-cli') {
-            sh "gpg --verify '${ env.JENKINS_ASC }' '${ env.JENKINS_WAR }'"
-            sh "unzip -qc '${ env.JENKINS_WAR }' META-INF/MANIFEST.MF | grep 'Jenkins-Version' | awk \'{print \$2}\'"
-            sh "az storage container upload \
-              --acount-name '${ env.AZURE_STORAGE_ACCOUNT }' \
-              --share-name '${ env.AZURE_SHARE_NAME }' \
-              --path '${ env.VERSION/jenkins.war }' \
+            sh 'gpg --verify "$JENKINS_ASC" "$JENKINS_WAR"'
+            sh '''
+              unzip -qc "$JENKINS_WAR" META-INF/MANIFEST.MF | grep 'Jenkins-Version' | awk '{print $2}'
```

Comment:
More readable I think.

I am keeping _shell_ quotes around anything that might include the workspace path, for the unlikely case that the agent FS root includes a space in its name. If you are running in K8S with a predictable environment you could probably drop them for readability, though.